core/services/wifi/main.py

```diff
@@ -167,8 +172,8 @@ async def disconnect() -> Any:
 
 @app.get("/hotspot", summary="Get hotspot state.")
 @version(1, 0)
-def hotspot_state() -> Any:
-    return wifi_manager.hotspot.is_running()
+def hotspot_state() -> HotspotStatus:
```

Comment:
This is an API break, a v2 field should be used or a new endpoint.

Reply:
good catch. I updated it to use a new endpoint